### ISSUE TITLE
tx1-2: Winners, groups of winners and OrganisationGroup

### ISSUE THREAD

**Issue by costezki** [OP-TED/ted-rdf-mapping#245]
V.2.2.6  The Organisation Group comprises organisations but as it is not linked to a lot or contract when the winners are being taken into consideration it is not possible to know to which lot/contract the group belongs to.  See 287956-2021
Answer: the grouping of Winners was re-discussed yesterday (2022-11-22) in the ontology WG.  The conclusion was that no mapping will ever use OrganisationGroup, but always provide individual Winners (as it is not possible to know or interpret any grouping). Thus even if there is a field in the input XML stipulating that the award is done to a group, this will be ignored. 

Note: This change has a significant impact on the handling of Organisations and OrganisationGroups as they are treated in an uniform manned for ALL of the roles. An additional confirmation is necessary if this is to be implemented in the future (with Winner being an exception to the general rule).  
Impact and a final implementation decision shall be further discussed in an EPO WG meeting.

This is tightly related to #111

**Comment by csnyulas** [comment]
BTW, the two commits mentioning this issue above are related to #242, not this issue.

**Comment by muricna** [comment]
All that is needed is that the Winners are not mapped to OrganisationGroup, we map just to the organisations  and that we remark that V.2.2.6 is not mappable or am I missing something?  I think this was implemented in the output but not in the conceptual mapping.

**Comment by costezki** [comment]
![image](https://user-images.githubusercontent.com/13567133/219333588-def32b8f-cab9-4a4e-a5a4-8d17ea7738c7.png)

**Comment by csnyulas** [comment]
This has been addressed by commenting out all the mappings related to creating the ContractorOrganisationGroup (see related commits).

Also updated CM (v.6.6.6) by removing the mapping (class/property paths) for fields `V.2.2.6` and `VII.1.6.3`, and adding explanation notes for both. We also removed the entire conceptual mapping line for field `V.2.2.6` for form F13, which doesn't have the concept of group of winners.

**Comment by muricna** [comment]
I do not think that it is enough to comment out the mappings they should be removed.
